readme.md

```diff
@@ -5,9 +5,33 @@
 ![GitHub contributors](https://img.shields.io/github/contributors/bitaxeorg/esp-miner)
 ![Alt](https://repobeats.axiom.co/api/embed/70889479b1e002c18a184b05bc5cbf2ed3718579.svg "Repobeats analytics image")
 
-# ESP-Miner
+# ESP-Miner with Voltage Monitoring
```

Comment:
This is no longer going to be in a fork. Please take that into consideration with the docs. Make the context correct.